src/ExtensionsSample/Samples/NotificationHubSamples.cs

```diff
@@ -49,8 +80,8 @@ public static class NotificationHubSamples
                 };
         }
 
-        //   The binding creates a strongly-typed AsyncCollector, which is used to send push notifications. 
-        //   The binding does not do anything with the results when the function exits.  
+        // This binding creates a strongly-typed AsyncCollector, which is used to send push notifications. 
+        // The binding does not do anything with the results when the function exits.  
```

Comment:
this "the binding does not do anything with the results" comment is confusing - I don't understand it, and I doubt users will either :) remcommend removing it here and elsewhere